packages/apps-config/src/ui/logos/index.ts

```diff
@@ -73,6 +73,7 @@ import mybank from './nodes/mybank.png';
 import nodeNFTMart from './nodes/nftmart.png';
 import nodeNodle from './nodes/nodle.svg';
 import oakTestnet from './nodes/oak-testnet.png';
+import nodeOriginTrail from './nodes/origintrail.png';
```

Comment:
Move up. (Under O)

Reply:
I think it is at the right place. Please let me know where to move it if necessary